src/ngAria/aria.js

```diff
@@ -377,6 +377,27 @@ ngAriaModule.directive('ngShow', ['$aria', function($aria) {
             });
           }
         }
+
+        if ($aria.config('bindKeypress') && isNodeOneOf(elem, ['A', 'BUTTON'])) {
+          elem.on('keypress', function(event) {
+            var keyCode = event.which || event.keyCode;
+
+            if (elem[0].nodeName === 'A') {
+              var hasHref = elem.attr('href') != null && elem.attr('href') !== '';
+              if ((keyCode === 13 && !hasHref) || keyCode === 32) {
```

Comment:
The browser handles ENTER for anchor. We just need to handle SPACE (and I would still check that it has no href).